spec/plainmonthday.html

```diff
@@ -392,7 +392,9 @@ <h1>ToTemporalMonthDay ( _item_ [ , _constructor_ [ , _options_ ] ] )</h1>
           1. Throw a *RangeError* exception.
         1. Let _calendar_ be ? ToOptionalTemporalCalendar(_result_.[[Calendar]]).
         1. Set _result_ to ? RegulateMonthDay(_result_.[[Month]], _result_.[[Day]], _overflow_).
-        1. Return ? CreateTemporalMonthDayFromStatic(_constructor_, _result_.[[Month]], _result_.[[Day]], _referenceISOYear_, _calendar_).
+        1. Set _result_ to ? CreateTemporalMonthDay(_result_.[[Month]], _result_.[[Day]], _referenceISOYear_, _calendar_).
+        1. Let _canonicalizeMonthDayOptions_ be ! OrdinaryObjectCreate(%Object.prototype%).
```

Comment:
this is a somewhat odd name for what's essentially `_emptyOptions_`

Reply:
I've changed it to canonicalMonthDayOptions as we discussed on chat. The reasoning was that I was trying to give a descriptive name that would suggest why this extra call to MonthDayFromFields was necessary.